pkgs/development/python-modules/deap/default.nix

```diff
@@ -9,19 +18,18 @@ buildPythonPackage rec {
     hash = "sha256-zAHemJLfp9G8mAPasoiS/q0XfwGCyB20c2CiQOrXeP8=";
   };
 
-  propagatedBuildInputs = [ numpy matplotlib ];
+  propagatedBuildInputs = [ matplotlib numpy pytest ];
```

Comment:
Could you please use `pytestCheckHook` instead?
and pytestCheckHook should go to `nativeCheckInputs`.

Reply:
Could you share the rationale? I'm unaware of the difference.